server/app/views/style/BaseStyles.java

```diff
@@ -90,7 +91,8 @@ public final class BaseStyles {
       StyleUtils.joinStyles(BaseStyles.FORM_LABEL_TEXT_COLOR, "text-base");
 
   /** For use on an `input` of type "checkbox". */
-  public static final String CHECKBOX = StyleUtils.joinStyles("h-4", "w-4", "mr-4", "align-middle");
+  public static final String CHECKBOX =
+      StyleUtils.joinStyles("h-4", "w-4", "mr-4", "align-middle", "self-center", "flex-none");
```

Comment:
"flex-none" keeps the checkboxes/radio buttons the same size when the text expands.